code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -3033,61 +2714,52 @@
 	name = "Boonie Bars"
 	desc = "Two delicious bars of minty chocolate. <i>\"Sometimes things are just... out of reach.\"</i>"
 	icon_state = "boonie"
-	bitesize = 1 //Two bars
+	bitesize = 2 //Two bars
 	wrapper = /obj/item/trash/boonie
-
-	New()
-		..()
-		reagents.add_reagent("nutriment", 4)
-		reagents.add_reagent("coco", 4)
+	list_reagents = list("nutriment" = 4, "coco" = 4)
+	tastes = list("peppermint" = 3, "falling into the sun" = 1)
 
 /obj/item/reagent_container/food/snacks/wrapped/chunk
 	name = "CHUNK box"
 	desc = "A bar of \"The <b>CHUNK</b>\" brand chocolate. <i>\"The densest chocolate permitted to exist according to federal law. We are legally required to ask you not to use this blunt object for anything other than nutrition.\"</i>"
 	icon_state = "chunk"
 	force = 10 //LEGAL LIMIT OF CHOCOLATE
-	bitesize = 2
+	bitesize = 3
 	wrapper = /obj/item/trash/chunk
-
-	New()
-		..()
-		reagents.add_reagent("nutriment", 5)
-		reagents.add_reagent("coco", 10)
+	list_reagents = list("nutriment" = 5, "coco" = 10)
+	tastes = list("compressed matter" = 1)
 
 /obj/item/reagent_container/food/snacks/wrapped/barcardine
 	name = "Barcardine Bars"
 	desc = "A bar of chocolate, it smells like the medical bay. <i>\"Chocolate always helps the pain go away.\"</i>"
 	icon_state = "barcardine"
 	wrapper = /obj/item/trash/barcardine
+	list_reagents = list("nutriment" = 3, "coco" = 3, "tramadol" = 1)
```

Comment:
Put a tiny bit of bicaridine to make it name-compliant?

Reply:
that might be a little extra powergamey but ok.

Reply:
You'd need 15 bars to get as much bicard as a single pill would give you. You can't eat that much even if you are starving, and if you try to get close to that you'll end up fat and slowed down by over-nutrition.

Really, as powergame materials go, this is as low as it can be.